editor/README.md

```diff
@@ -1,136 +1,41 @@
-**This** is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-## Quick start
+# Vision Zero Editor (VZE)
 
-1. Start your database and graphql engine
+![Vision Zero Editor](../docs/images/editor-crash-details.png)
 
-2. Use the `env_template` to configure the local environment variables. These can be copied from your existing VZE `.env` file, by replacing `REACT_APP_` with `NEXT_PUBLIC_` in the variable name:
+The Vision Zero Editor (VZE) enables City of Austin staff to view and edit traffic crash records. It's the central application that supports the City's [Vision Zero program](https://www.austintexas.gov/department/vision-zero) and related safety efforts carried out by the Transportation & Public Works Department.
 
-```shell
-# run this cmd and then edit .env.local as needed
-cp -n env_template .env.local
-```
+- Production: https://visionzero.austin.gov/editor
+- Staging: https://visionzero-staging.austinmobility.io/editor
 
-3. Activate your `node` environment (`v20` is required):
+## Quick start
+
+1. Configure the environment. Create a file called `.env.local`  here, in the root of the `/editor` subdirectory. Save the contents of the 1Password item called **Vision Zero Editor (VZE) Environment File** into this file.
```

Comment:
What do y'all think about specifying that the editor can be run locally without running the db? Thats still the case right, it can point to the staging db?

Reply:
I have always felt a little iffy on encouraging this pattern, since it's easy to forget which backend you're looking at, and you can even potentially introduce unexpected data into the staging DB.